backend/server.js

```diff
@@ -1,8 +1,8 @@
 import express from "express";
-import { sequelize } from "./database.js";
-import { Stock } from "./models/stock.js";
 import cors from "cors";
 import morgan from "morgan";
+import { sequelize } from "./database.js";
```

Comment:
It seems like sequelize is not needed in this file anymore

Reply:
At the very bottom of server.js there is a line of code that uses sequelize. I'm pretty sure it updates the database when things change.